persist-cli/src/main/java/io/ballerina/persist/utils/SqlScriptGenerationUtils.java

```diff
@@ -0,0 +1,420 @@
+            }
+            if (!uniqueKeyScript.equals(UNIQUE_KEY_START_SCRIPT)) {
+                script = script.concat(uniqueKeyScript.substring(0, uniqueKeyScript.length() - 2).concat("),"));
+            }
+            uniqueKeyScript = UNIQUE_KEY_START_SCRIPT;
+        }
+        return script;
+    }
+
+    private static String getReferenceAction(String value) {
+        switch (value) {
+            case RESTRICT:
+                return RESTRICT_SYNTAX;
+            case CASCADE:
+                return CASCADE_SYNTAX;
+            case NO_ACTION:
+                return NO_ACTION_SYNTAX;
+            case SET_NULL:
+                return SET_NULL_SYNTAX;
+            default:
+                return SET_DEFAULT_SYNTAX;
+        }
+    }
+
+    private static String eliminateDoubleQuotes(String text) {
+        return text.substring(1, text.length() - 1);
+    }
+
+    private static String getType(String type) {
+        switch (type) {
+            case PersistToolsConstants.BallerinaTypes.INT:
+                return PersistToolsConstants.SqlTypes.INT;
+            case PersistToolsConstants.BallerinaTypes.BOOLEAN:
+                return PersistToolsConstants.SqlTypes.BOOLEAN;
+            case PersistToolsConstants.BallerinaTypes.DECIMAL:
+                return PersistToolsConstants.SqlTypes.DECIMAL;
+            case PersistToolsConstants.BallerinaTypes.FLOAT:
+                return PersistToolsConstants.SqlTypes.FLOAT;
+            case PersistToolsConstants.BallerinaTypes.DATE:
+                return PersistToolsConstants.SqlTypes.DATE;
+            case PersistToolsConstants.BallerinaTypes.TIME_OF_DAY:
+                return PersistToolsConstants.SqlTypes.TIME;
+            case PersistToolsConstants.BallerinaTypes.UTC:
+                return PersistToolsConstants.SqlTypes.TIME_STAMP;
+            case PersistToolsConstants.BallerinaTypes.CIVIL:
+                return PersistToolsConstants.SqlTypes.DATE_TIME;
+            case PersistToolsConstants.BallerinaTypes.STRING:
+                return PersistToolsConstants.SqlTypes.VARCHAR;
+            default:
+                return null;
+        }
+    }
+
+    private static void createSqlFile(String script,
+                                     String tableName, HashMap<String, List<String>> referenceTables,
+                                     List<String> tableNamesInScript, Path directoryPath) throws BalException {
+        try {
+            String content = EMPTY;
+            Path filePath = Paths.get(String.valueOf(directoryPath), PersistToolsConstants.FILE_NAME);
+            if (Files.exists(filePath)) {
+                byte[] bytes = Files.readAllBytes(filePath);
+                content = new String(bytes, StandardCharsets.UTF_8);
+                String tableNames = "";
+                int firstIndex = 0;
+                if (referenceTables.containsKey(tableName)) {
```

Comment:
Why are we reading the files again? These sql commands should be in-memory if we havent fully processed it.